### ISSUE TITLE
unknown error: Chrome version must be >= 55.0.2883.0

### ISSUE THREAD

**Issue by vsadineni1** [appium/appium#8809]
## The problem

I am testing a hybrid app on Android Emulator 6.0 Nexus_5x_API_23. When the test tries to swicth 
to the web context, I get the below error:

**"unknown error: Chrome version must be >= 55.0.2883.0\n  (Driver info: chromedriver=2.28.455506 (18f6627e265f442aeec9b6661a49fe819aeeea1f),platform=Linux 4.4.0-83-generic x86_64)"}**

I tried to pass the chromedriver exe as part of the appium command as below:
**_appium -p 7777 --avd Nexus_5X_API_23 --chromedriver-executable /home/myname/Downloads/2.28/chromedriver_**

## Environment

* Appium version (or git revision) that exhibits the issue: 1.6.4
* Last Appium version that did not exhibit the issue (if applicable): N/A
* Desktop OS/version used to run Appium: Ubntu 16.4
* Node.js version (unless using Appium.app|exe):
* Mobile platform/version under test: Android 6.0  
* Real device or emulator/simulator: Emulator
* Appium CLI or Appium.app|exe: 

## Details

I tried with the latest chrome driver version: 2.9
also tried with the chrome driver version: 2.28

## Link to Appium logs

https://gist.github.com/vsadineni1/b41c801a8df720708e9d1324e81cc32f

**Comment by mykola-mokhnach** [comment]
Please check https://github.com/appium/appium/blob/master/docs/en/advanced-concepts/chromedriver.md

**Comment by vsadineni1** [comment]
@mykola-mokhnach Thank you for your response.

Yes, I have tried all the below versions of chrome driver. **But no luck**. **I am using Android 6.0 Nexus_5x_API_23 emulator. 
Any suggestion on which version will work for this?**

------
appium -p 7777 --avd Nexus_5X_API_23 --chromedriver-executable /home/venkata.sadineni/Downloads/2.23/chromedriver

Caused by: org.openqa.selenium.WebDriverException: An unknown server-side error occurred while processing the command. Original error: unknown error: Chrome version must be >= 51.0.2704.0
(Driver info: chromedriver=2.23.409687 (c46e862757edc04c06b1bd88724d15a5807b84d1),platform=Linux 4.4.0-83-generic x86_64) (WARNING: The server did not provide any stacktrace information)

------
appium -p 7777 --avd Nexus_5X_API_23 --chromedriver-executable /home/venkata.sadineni/Downloads/2.28/chromedriver

org.openqa.selenium.WebDriverException: An unknown server-side error occurred while processing the command. Original error: unknown error: Chrome version must be >= 55.0.2883.0
(Driver info: chromedriver=2.28.455506 (18f6627e265f442aeec9b6661a49fe819aeeea1f),platform=Linux 4.4.0-83-generic x86_64) (WARNING: The server did not provide any stacktrace information)

-------
appium -p 7777 --avd Nexus_5X_API_23 --chromedriver-executable /home/venkata.sadineni/Downloads/2.29/chromedriver

org.openqa.selenium.WebDriverException: An unknown server-side error occurred while processing the command. Original error: session not created exception: Chrome version must be >= 56.0.2884.0
(Driver info: chromedriver=2.29.461571 (8a88bbe0775e2a23afda0ceaf2ef7ee74e822cc5),platform=Linux 4.4.0-83-generic x86_64) (WARNING: The server did not provide any stacktrace information)
----------
appium -p 7777 --avd Nexus_5X_API_23 --chromedriver-executable /home/venkata.sadineni/Downloads/2.30/chromedriver

org.openqa.selenium.WebDriverException: An unknown server-side error occurred while processing the command. Original error: session not created exception: Chrome version must be >= 58.0.3029.0
(Driver info: chromedriver=2.30.477691 (6ee44a7247c639c0703f291d320bdf05c1531b57),platform=Linux 4.4.0-83-generic x86_64) (WARNING: The server did not provide any stacktrace information)
----

**Comment by mykola-mokhnach** [comment]
2.23 seems to be too old, but 2.28 is too new. Try something in between - there is a table in that document, which should help you to select a correct version.

**Comment by vsadineni1** [comment]
I have tried all the versions between 2.23 & 2.28. I get the same error as shown below.
---
appium -p 7777 --avd Nexus_5X_API_23 --chromedriver-executable /home/venkata.sadineni/Downloads/2.24/chromedriver

An unknown server-side error occurred while processing the command. Original error: unknown error: Chrome version must be >= 52.0.2743.0
(Driver info: chromedriver=2.24.417424 (c5c5ea873213ee72e3d0929b47482681555340c3),platform=Linux 4.4.0-83-generic x86_64) (WARNING: The server did not provide any stacktrace information)

---
appium -p 7777 --avd Nexus_5X_API_23 --chromedriver-executable /home/venkata.sadineni/Downloads/2.25/chromedriver

An unknown server-side error occurred while processing the command. Original error: unknown error: Chrome version must be >= 53.0.2785.0
(Driver info: chromedriver=2.25.426924 (649f9b868f6783ec9de71c123212b908bf3b232e),platform=Linux 4.4.0-83-generic x86_64) (WARNING: The server did not provide any stacktrace information)
---
appium -p 7777 --avd Nexus_5X_API_23 --chromedriver-executable /home/venkata.sadineni/Downloads/2.26/chromedriver

An unknown server-side error occurred while processing the command. Original error: unknown error: Chrome version must be >= 53.0.2785.0
(Driver info: chromedriver=2.26.436382 (70eb799287ce4c2208441fc057053a5b07ceabac),platform=Linux 4.4.0-83-generic x86_64) (WARNING: The server did not provide any stacktrace information)
---
appium -p 7777 --avd Nexus_5X_API_23 --chromedriver-executable /home/venkata.sadineni/Downloads/2.27/chromedriver

An unknown server-side error occurred while processing the command. Original error: unknown error: Chrome version must be >= 54.0.2840.0
(Driver info: chromedriver=2.27.440175 (9bc1d90b8bfa4dd181fbbf769a5eb5e575574320),platform=Linux 4.4.0-83-generic x86_64) (WARNING: The server did not provide any stacktrace information)
---

One dumb question I have is: **what is the version of the Chrome browser on Android Emulator 6.0 Nexus_5X_API_23?**

I tried to find this by following the steps: Launch browser -> Settings. But I could not find the browser version in any of the sections.

I googled around. It seems like the version os: Chrome 51 as per the below link:
https://www.whatismybrowser.com/developers/tools/user-agent-parser/browse/view/chrome--630
So as per the table: Chrome driver version: 2.23 should work. But no luck with this one.
